sdk/src/Services/SecurityToken/Custom/SAML/ADFSAuthenticationHandlers.cs

```diff
@@ -157,6 +244,31 @@ private static string QueryProvider(Uri identityProvider, IWebProxy proxySetting
             return responseData;
         }
 
+        private static HttpWebRequest CreateRequest(IWebProxy proxySettings, ICredentials credentials, string authenticationType, Uri uri, CookieContainer cookieContainer, string connectionGroup)
+        {
+#pragma warning disable SYSLIB0014 // Disable obsolete warning for WebRequest.Create because we can't switch to HttpClient while we still target .NET Framework 3.5
```

Comment:
Is this `pragma warning` still needed? There shouldn't be a 3.5 target anymore.

Reply:
I got a Error warning after removing  `pragma warning`. Hence, didn't remove it for now.